src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -152,7 +161,10 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, Component
         /** Initialize local constants **/
         this.showExistingQuestions = false;
         this.showExistingQuestionsFromCourse = true;
+        this.showExistingQuestionsFromExam = false;
+        this.showExistingQuestionsFromFile = false;
```

Comment:
You don't need to initialize again if you did that above

Reply:
👍